invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitySubmission/RecordCommunitySubmissionModal.js

```diff
@@ -34,15 +34,19 @@ export class RecordCommunitySubmissionModal extends Component {
     return userMembership && community?.access.review_policy === "open";
   };
 
+  wasPublished = (requestData) => {
```

Comment:
Let's keep the same naming as we use around.
As the permission name:
```suggestion
  isIncludedDirectly = (requestData) => {
```
or `isDirectlyPublished` as the first submission.